docs/storage/azure-storage-components.md

```diff
@@ -62,6 +70,34 @@ You also need to assign the following roles to the user account you are logged i
 
 :::zone-end
 
+:::zone pivot="azure-developer-cli"
+
+The [Azure Developer CLI](/azure/developer/azure-developer-cli/overview) enables you to provision and deploy Azure resources using a template system. This tutorial provides a [complete template](https://github.com/Azure-Samples/dotnet-aspire-connect-storage.git) that provisions the required Azure resources and includes the completed sample application code.
+
+1. Run the following commands to initialize and run the template:
+
+    ```azurecli
```

Comment:
We had recent feedback about ensuring that CLI commands are easy to copy and paste into terminal sessions, with that said, we should ensure that these commands are either properly newline delimited so that they can be copied/pasted as-is, or have each command be on its own. For more info, see #409.

Reply:
Thanks for this feedback, I switched the azd commands to individual boxes so that I could add a description for each one for users still newer to azd